x-pack/plugins/maps/public/components/layer_panel/join_editor/resources/_join.scss

```diff
@@ -1,8 +1,14 @@
 .mapJoinItem {
+
+  position: relative;
   background: tintOrShade($euiColorLightShade, 85%, 0);
   border-radius: $euiBorderRadius;
   padding: $euiSizeXS;
-  position: relative;
+
+  .mapJoinItem__inner {
+    overflow: auto;
```

Comment:
Might want to just say `overflow-x: auto` so in stupid browsers you don't get a vertical scrollbar unnecessarily.